pkg/reader/notify/notify.go

```diff
@@ -11,3 +16,12 @@ type Event interface {
 	SetProcess(*tetragon.Process)
 	Encapsulate() tetragon.IsGetEventsResponse_Event
 }
+
+func EventTypeString(event Event) string {
+	// Get the concrete type of the event
+	ty := fmt.Sprintf("%T", event)
```

Comment:
Maybe an alternative way to get the type of event is `reflect.TypeOf(event).String()`. I don't know if there are any reason to prefer one way over another but this seems to work fine.

Reply:
@tpapagian from my understanding they are basically equivalent. maybe one is more idiomatic than the other, who knows 😀

Reply:
According to some Go experts I asked, the way I currently have it does indeed seem to be preferred